tests/test_full_library.py

```diff
@@ -234,22 +237,21 @@ def test_full_lib(tmpdir):
 
         # Step 4: Generate CPP code
```

Comment:
```suggestion
        os.chdir("..")
        # Step 4: Generate CPP code
```

Reply:
Might also work if you change the relative filenames in the following steps and remove "package", but that is how I did it on my branch.

Reply:
@poshul Any updates on this? If you just accept the change, I can check in travis if it was enough.